BridgeCareApp/BridgeCare/Services/SummaryReport/BridgeRateDeckAreaWorkSummary.cs

```diff
@@ -0,0 +1,184 @@
+﻿using BridgeCare.Models;
+using OfficeOpenXml;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Web;
+
+namespace BridgeCare.Services
+{
+    public static class BridgeRateDeckAreaWorkSummary
+    {
+        #region constants
+        const string BridgeCare = "Bridge Care";
+        const string Good = "Good";
+        const string Fair = "Fair";
+        const string Poor = "Poor";
+        #endregion
+
+        /// <summary>
+        /// Fill work summary bridge poor on off rate and deck area sections
+        /// </summary>
+        /// <param name="worksheet"></param>
+        /// <param name="currentCell"></param>
+        /// <param name="simulationYears"></param>
+        /// <param name="simulationDataModels"></param>
+        public static void FillBridgeRateDeckAreaWorkSummarySections(ExcelWorksheet worksheet, CurrentCell currentCell, List<int> simulationYears, List<SimulationDataModel> simulationDataModels)
+        {
+            FillPoorBridgeOnOfRateSection(worksheet, currentCell, simulationYears, simulationDataModels);
```

Comment:
Should this be 'FillPoorBridgeOnOffRateSection'?

Reply:
Renamed, thanks.